07_cmd_lab/cmdlab.txt

```diff
@@ -3,11 +3,26 @@ Questions online: https://www.tocode.co.il/bundles/unix1/lessons/cmdlab
 
 1. Login to your UNIX account. How do you find:
   - your current shell?
+	
+	echo $0 / which $0
+	
   - your default shell?
+  
+	echo @SHELL
+  
   - a list of all shells currently installed on the machine?
+  
+	cat /etc/shells
 
 2. View the file /etc/passwd on your machine with the command "cat /etc/passwd". Then read about the file in "man 5 passwd". What information is held in the file? What are the fields in each line?
 
+	the information is all the user acounts + password in the OS
+	the fields are (seperated by ":") login name, encrypted password, user ID, group ID, user name, home directory and user interpreter 
+
 3. In Unix, the /etc folder holds configuration files. Display all files in "/etc" folder ordered by file size (Hint: search in "man ls")
+	
+	ll -S
```

Comment:
sweet :) btw bonus question what is ```ll``` and where it's defined?
hint: it's not a binary :)